content/ja/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -93,6 +93,7 @@ kubectl edit SampleDB/example-database # 手動でいくつかの設定を変更
   * [Kopf](https://github.com/nolar/kopf) (Kubernetes Operator Pythonic Framework)
   * [kube-rs](https://kube.rs/) (Rust)
   * [KUDO](https://kudo.dev/)（Kubernetes Universal Declarative Operator）を使います
+  * [Mast](https://docs.ansi.services/mast/user_guide/operator/)
```

Comment:
Can you move this link above the Metacontroller link because current update breaks alphabetical order?

Reply:
Done